fastly.tf

```diff
@@ -1,9 +1,10 @@
 module "fastly" {
-  source = "github.com/mergermarket/tf_fastly_frontend"
+  source = "git::https://github.com/mergermarket/tf_fastly_frontend.git"
```

Comment:
@grahamlyons yes - this format of source lets you easy add branch / tag you want to pin to